### ISSUE TITLE
protontricks not launching on steam deck

### ISSUE THREAD

**Issue by steamdecklover** [Matoking/protontricks#284]
**Describe the bug**
whenever trying to launch protontricks,it gives an error message

**To Reproduce**
1. launch protontricks from discover 
2.  launch fails and error is displayed

**Expected behavior**
it launches so i can install uplay for ghost recon breakpoint

**System (please complete the following information):**
 - Distro: SteamOS/Arch
 - Protontricks installation method: flatpak
 - Protontricks version: 1.11
 - Steam vers

**Additional context**
Protontricks was closed due to the following error:

Traceback (most recent call last):
  File "/app/lib/python3.11/site-packages/protontricks/cli/util.py", line 159, in wrapper
    return cli_func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/lib/python3.11/site-packages/protontricks/cli/main.py", line 244, in main
    steam_apps = get_steam_apps(
                 ^^^^^^^^^^^^^^^
  File "/app/lib/python3.11/site-packages/protontricks/steam.py", line 1329, in get_steam_apps
    steam_apps += get_custom_compat_tool_installations(steam_root=steam_root)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/lib/python3.11/site-packages/protontricks/steam.py", line 1074, in get_custom_compat_tool_installations
    for tool_app in get_custom_compat_tool_installations_in_dir(dir_):
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/lib/python3.11/site-packages/protontricks/steam.py", line 1014, in get_custom_compat_tool_installations_in_dir
    compat_tools = compat_tools["compatibilitytools"]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^
KeyError: 'compatibilitytools'


=============

Please include this entire error message when making a bug report.
Log messages:

Running inside Flatpak sandbox, version 1.14.4.
Found Steam directory at /home/deck/.local/share/Steam
Using default Steam Runtime at /home/deck/.local/share/Steam/ubuntu12_32/steam-runtime
WINETRICKS environment variable is not available. Searching from $PATH.
Found 2 Steam library folders

**Comment by Matoking** [comment]
I released Protontricks 1.11.1 which should be available in Flathub within a few hours. Once the update is available, could you check if it solves this issue?

**Comment by steamdecklover** [comment]
worked for me thx
